CHANGELOG.rst

```diff
@@ -4,13 +4,25 @@ CHANGELOG
 This document describes changes between each past release.
 
 
-5.0.0 (unreleased)
+5.0.0 (2016-05-12)
 ==================
 
 **Breaking changes**
 
 - Rename the last_modified client parameter into if_match (#68)
 
+**New features**
+
+- Display a better message when having 403 on create_collection and
+  create_record methods (#49)
+- Add get_endpoints as a feature of the public_api (#60)
+- Add a server_info methods to ease capability checking (#70)
```

Comment:
I would not mention capability here, it can also be used for many other things (user id, settings, etc.). We could just say to retrieve the root url info